### PR TITLE
Create design decisions document

### PR DESCRIPTION
**Pull request by javsalgar**
In order to improve the transparency as well as making the onboarding easier for contributors,
this PR creates a design decisions document. As several changes are coming in the following weeks, we
expect updates to this document very soon.

### CODE REVIEW

**Review by andresbono — COMMENTED** [review]
I'd like to add some other suggestions:

- Put related issues and PRs (if any)
- Start every decision with a TL;DR

I guess these 2 suggestions don't make sense yet, I mean, the first decision is so simple that doesn't need a TL;DR, but it would be nice for the following ones.

File: DESIGN_DECISIONS.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Bitnami Helm charts design decisions
+
+The following document is meant to log all reasoning and design decisions with respect
+to the Bitnami Helm charts.
```
Comment (carrodher) [inline DESIGN_DECISIONS.md:4]: It is not needed to break the lines
```suggestion
The following document is meant to log all reasoning and design decisions with respect to the Bitnami Helm charts.
```

File: DESIGN_DECISIONS.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  TL;DR: <brief summary> 
+
+  <Reasoning and explanation, putting any related issues and PRs (if any)>
+```
+
+## 2021-01-19: Creation of the design decisions document
+
+TL;DR: In order to improve the transparency as well as making the onboarding easier for contributors,
+we decided to create this document.
```
Comment (carrodher) [inline DESIGN_DECISIONS.md:19]:
```suggestion
TL;DR: In order to improve the transparency as well as making the onboarding easier for contributors, we decided to create this document.
```

File: DESIGN_DECISIONS.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+The following document is meant to log all reasoning and design decisions with respect
+to the Bitnami Helm charts.
+
+Use the following format for contributing to this document:
+
+```
+  ## YYYY-MM-DD: <Title for the design decision>
+
+  TL;DR: <brief summary>
```
Comment (andresbono) [inline DESIGN_DECISIONS.md:10]: What if we bold `TL;DR:`? The same thing for `PR:` below.

Comment (javsalgar) [reply DESIGN_DECISIONS.md]: Done

**Review by alemorcuq — APPROVED** [review]
LGTM. Perhaps it's better to update the date to today, though.

**Review by carrodher — APPROVED** [review]
LGTM

**Review by andresbono — APPROVED** [review]
